Configuration/TCA/Overrides/tt_content.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+defined('TYPO3_MODE') || die('Access denied.');
```

Comment:
Perhaps `TYPO3` should be used here as `TYPO3_MODE` is deprecated since version 11.0 and `TYPO3` is available since version 10.4.11 already: https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/11.0/Deprecation-92947-DeprecateTYPO3_MODEAndTYPO3_REQUESTTYPEConstants.html ?

Reply:
thanks, you are right, i will chage after rebasing